diff --git a/doc.ddoc b/doc.ddoc
index 4e47ad3..e1ed20c 100644
--- a/doc.ddoc
+++ b/doc.ddoc
@@ -10,7 +10,6 @@ _=
 NAVIGATION=
 $(DIVID cssmenu, $(UL
     $(MENU index.html, D $(LATEST))
-    $(MENU download.html, $(B Download))
     $(MENU getstarted.html, Getting Started)
     $(MENU changelog.html, Change Log)
     $(MENU_W_SUBMENU D Reference)
diff --git a/posix.mak b/posix.mak
index b6cc187..32afe01 100644
--- a/posix.mak
+++ b/posix.mak
@@ -207,7 +207,7 @@ $(DOC_OUTPUT_DIR)/dmd-%.verbatim : %.ddoc dcompiler.dd verbatim.ddoc $(DMD)
 # Rulez
 ################################################################################
 
-all : docs html
+all : html druntime-release phobos-release d.tag
 
 docs : phobos-prerelease druntime-prerelease druntime-release phobos-release	\
 	apidocs-release apidocs-prerelease
@@ -233,9 +233,9 @@ ${GENERATED}/${LATEST}.ddoc :
 	mkdir -p $(dir $@)
 	echo "LATEST=${LATEST}" >$@
 
-${GENERATED}/modlist-${LATEST}.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR)-$(LATEST)/.cloned $(PHOBOS_DIR)-$(LATEST)/.cloned
+${GENERATED}/modlist-${LATEST}.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR) $(PHOBOS_DIR)
 	mkdir -p $(dir $@)
-	$(STABLE_RDMD) modlist.d $(DRUNTIME_DIR)-$(LATEST) $(PHOBOS_DIR)-$(LATEST) $(MOD_EXCLUDES_RELEASE) >$@
+	$(STABLE_RDMD) modlist.d $(DRUNTIME_DIR) $(PHOBOS_DIR) $(MOD_EXCLUDES_RELEASE) >$@
 
 ${GENERATED}/modlist-prerelease.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR)/.cloned $(PHOBOS_DIR)/.cloned
 	mkdir -p $(dir $@)
@@ -331,8 +331,8 @@ dlangspec.verbatim.txt : $(DMD) verbatim.ddoc dlangspec-consolidated.d
 # dmd compiler, latest released build and current build
 ################################################################################
 
-$(DMD) : ${DMD_DIR}/.cloned
-	${MAKE} --directory=${DMD_DIR}/src -f posix.mak -j 4
+#$(DMD) : ${DMD_DIR}/.cloned
+#	${MAKE} --directory=${DMD_DIR}/src -f posix.mak -j 4
 
 $(DMD_REL) : ${DMD_DIR}-${LATEST}/.cloned
 	${MAKE} --directory=${DMD_DIR}-${LATEST}/src -f posix.mak -j 4
@@ -346,9 +346,9 @@ druntime-prerelease : ${DRUNTIME_DIR}/.cloned $(DMD) $(STD_DDOC_PRE)
 		DOCDIR=${DOC_OUTPUT_DIR}/phobos-prerelease \
 		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC_PRE))"
 
-druntime-release : ${DRUNTIME_DIR}-${LATEST}/.cloned $(DMD_REL) $(STD_DDOC)
-	${MAKE} --directory=${DRUNTIME_DIR}-${LATEST} -f posix.mak target doc \
-	  DMD=$(DMD_REL) \
+druntime-release : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC)
+	${MAKE} --directory=${DRUNTIME_DIR} -f posix.mak target doc \
+	  DMD=$(DMD) \
 	  DOCDIR=${DOC_OUTPUT_DIR}/phobos \
 		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC))"
 
@@ -373,12 +373,12 @@ phobos-prerelease : ${PHOBOS_DIR}/.cloned $(STD_DDOC_PRE) druntime-prerelease
 	  STDDOC="$(addprefix `pwd`/, $(STD_DDOC_PRE))" \
 	  DOC_OUTPUT_DIR=${DOC_OUTPUT_DIR}/phobos-prerelease html -j 4
 
-phobos-release : ${PHOBOS_DIR}-${LATEST}/.cloned $(DMD_REL) $(STD_DDOC) \
+phobos-release : ${PHOBOS_DIR} $(DMD) $(STD_DDOC) \
 		druntime-release
-	${MAKE} --directory=${PHOBOS_DIR}-${LATEST} -f posix.mak -j 4 \
+	${MAKE} --directory=${PHOBOS_DIR} -f posix.mak -j 4 \
 	  html \
-	  DMD=$(DMD_REL) \
-	  DRUNTIME_PATH=${DRUNTIME_DIR}-${LATEST} \
+	  DMD=$(DMD) \
+	  DRUNTIME_PATH=${DRUNTIME_DIR} \
 	  DOC_OUTPUT_DIR=${DOC_OUTPUT_DIR}/phobos \
 	  STDDOC="$(addprefix `pwd`/, $(STD_DDOC))"
 
diff --git a/std_navbar-release.ddoc b/std_navbar-release.ddoc
index 896f264..8750058 100644
--- a/std_navbar-release.ddoc
+++ b/std_navbar-release.ddoc
@@ -1,7 +1,6 @@
 NAVIGATION=
 $(DIVID cssmenu, $(UL
     $(MENU ../index.html, D Lib $(LATEST))
-    $(MENU ../phobos-prerelease/index.html, Prerelease Version)
     $(MODULE_MENU)
     $(MENU http://code.dlang.org, 3rd Party Packages)
 ))
