diff --git a/dlang.org.ddoc b/dlang.org.ddoc
index 67a01b6c..310c8237 100644
--- a/dlang.org.ddoc
+++ b/dlang.org.ddoc
@@ -231,7 +231,6 @@ $(DIVID cssmenu, $(UL
     $(MENU https://tour.dlang.org, Learn)
     $(MENU_W_SUBMENU_LINK $(ROOT_DIR)documentation.html, Documentation)
       $(NAVIGATION_DOCUMENTATION)
-    $(MENU $(ROOT_DIR)download.html, Downloads)
     $(MENU https://code.dlang.org, Packages)
     $(MENU_W_SUBMENU_LINK $(ROOT_DIR)community.html, Community)
       $(NAVIGATION_COMMUNITY)
diff --git a/posix.mak b/posix.mak
index f5c8824e..a469c094 100644
--- a/posix.mak
+++ b/posix.mak
@@ -60,10 +60,6 @@ PHOBOS_LATEST_DIR_GENERATED=$(GENERATED)/phobos-release
 #   Makefile dependencies and rules
 PHOBOS_FILES := $(shell find $(PHOBOS_DIR) -name '*.d' -o -name '*.mak' -o -name '*.ddoc')
 PHOBOS_FILES_GENERATED := $(subst $(PHOBOS_DIR), $(PHOBOS_DIR_GENERATED), $(PHOBOS_FILES))
-$(shell [ ! -d $(PHOBOS_DIR) ] && git clone --depth=1 ${GIT_HOME}/phobos $(PHOBOS_DIR))
-$(shell [ ! -d $(PHOBOS_LATEST_DIR) ] && git clone -b v${LATEST} --depth=1 ${GIT_HOME}/phobos $(PHOBOS_LATEST_DIR))
-PHOBOS_LATEST_FILES := $(shell find $(PHOBOS_LATEST_DIR) -name '*.d' -o -name '*.mak' -o -name '*.ddoc')
-PHOBOS_LATEST_FILES_GENERATED := $(subst $(PHOBOS_LATEST_DIR), $(PHOBOS_LATEST_DIR_GENERATED), $(PHOBOS_LATEST_FILES))
 ################################################################################
 
 # stable dub and dmd versions used to build dpl-docs
@@ -129,6 +125,15 @@ DDOC_VARS_LATEST_HTML=\
 	  DOCSRC="$(PWD)" \
 	  VERSION="$(abspath ${DMD_DIR}/VERSION)"
 
+DDOC_VARS_RELEASE_HTML=\
+	  DOC_OUTPUT_DIR="${DOC_OUTPUT_DIR}/phobos" \
+	  STDDOC="$(addprefix $(PWD)/, $(STD_DDOC))" \
+	  DMD="$(abspath $(DMD))" \
+	  DMD_DIR="$(abspath ${DMD_DIR})" \
+	  DRUNTIME_PATH="$(abspath ${DRUNTIME_DIR})" \
+	  DOCSRC="$(PWD)" \
+	  VERSION="$(abspath ${DMD_DIR}/VERSION)"
+
 DDOC_VARS=\
 	DMD="$(abspath ${DMD})" \
 	DMD_DIR="$(abspath ${DMD_DIR})" \
@@ -202,8 +207,7 @@ SPEC_ROOT=$(addprefix spec/, \
 	abi simd betterc)
 SPEC_DD=$(addsuffix .dd,$(SPEC_ROOT))
 
-CHANGELOG_FILES=changelog/${NEXT_VERSION}_pre \
-				$(basename $(subst _pre.dd,.dd,$(wildcard changelog/*.dd))) \
+CHANGELOG_FILES=$(basename $(subst _pre.dd,.dd,$(wildcard changelog/*.dd)))
 
 # Website root filenames. They have extension .dd in the source
 # and .html in the generated HTML. Save for the expansion of
@@ -232,7 +236,7 @@ ALL_FILES = $(ALL_FILES_BUT_SITEMAP) $(DOC_OUTPUT_DIR)/sitemap.html
 # Rulez
 ################################################################################
 
-all : docs html
+all : html dmd-release druntime-release phobos-release d.tag
 
 docs-release: dmd-release druntime-release phobos-release apidocs-release
 docs-prerelease: dmd-prerelease druntime-prerelease phobos-prerelease apidocs-prerelease
@@ -260,9 +264,9 @@ ${GENERATED}/${LATEST}.ddoc :
 	mkdir -p $(dir $@)
 	echo "LATEST=${LATEST}" >$@
 
-${GENERATED}/modlist-${LATEST}.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_LATEST_DIR) $(PHOBOS_LATEST_DIR) $(DMD_LATEST_DIR)
+${GENERATED}/modlist-${LATEST}.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR) $(PHOBOS_DIR) $(DMD_DIR)
 	mkdir -p $(dir $@)
-	$(STABLE_RDMD) modlist.d $(DRUNTIME_LATEST_DIR) $(PHOBOS_LATEST_DIR) $(DMD_LATEST_DIR) $(MOD_EXCLUDES_RELEASE) \
+	$(STABLE_RDMD) modlist.d $(DRUNTIME_DIR) $(PHOBOS_DIR) $(DMD_DIR) $(MOD_EXCLUDES_RELEASE) \
 		$(addprefix --dump , object std etc core ddmd) >$@
 
 ${GENERATED}/modlist-prerelease.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR) $(PHOBOS_DIR)
@@ -420,14 +424,14 @@ ${DMD_DIR} ${DRUNTIME_DIR} ${PHOBOS_DIR} ${TOOLS_DIR} ${INSTALLER_DIR}:
 # dmd compiler, latest released build and current build
 ################################################################################
 
-$(DMD) : ${DMD_DIR}
-	${MAKE} --directory=${DMD_DIR}/src -f posix.mak AUTO_BOOTSTRAP=1
+#$(DMD) : ${DMD_DIR}
+#	${MAKE} --directory=${DMD_DIR}/src -f posix.mak AUTO_BOOTSTRAP=1
 
 $(DMD_LATEST) : ${DMD_LATEST_DIR}
 	${MAKE} --directory=${DMD_LATEST_DIR}/src -f posix.mak AUTO_BOOTSTRAP=1
 
-dmd-release : $(STD_DDOC) $(DMD_LATEST_DIR) $(DMD_LATEST)
-	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_LATEST_DIR) -f posix.mak html $(DDOC_VARS_LATEST_HTML)
+dmd-release : $(STD_DDOC) $(DMD_DIR) $(DMD)
+	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html $(DDOC_VARS_RELEASE_HTML)
 
 dmd-prerelease : $(STD_DDOC_PRE) $(DMD_DIR) $(DMD)
 	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html $(DDOC_VARS_HTML)
@@ -451,8 +455,8 @@ druntime-prerelease : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC_PRE)
 		DOCDIR=${DOC_OUTPUT_DIR}/phobos-prerelease \
 		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC_PRE))"
 
-druntime-release : ${DRUNTIME_LATEST_DIR} $(DMD_LATEST) $(STD_DDOC)
-	${MAKE} --directory=${DRUNTIME_LATEST_DIR} -f posix.mak target doc $(DDOC_VARS_LATEST_HTML) \
+druntime-release : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC)
+	${MAKE} --directory=${DRUNTIME_DIR} -f posix.mak target doc $(DDOC_VARS_RELEASE_HTML) \
 	  DOCDIR=${DOC_OUTPUT_DIR}/phobos \
 	  DOCFMT="$(addprefix `pwd`/, $(STD_DDOC))"
 
@@ -475,9 +479,9 @@ ${DOC_OUTPUT_DIR}/phobos-prerelease/object.verbatim : $(DMD)
 phobos-prerelease : ${PHOBOS_FILES_GENERATED} $(STD_DDOC_PRE) druntime-prerelease
 	$(MAKE) --directory=$(PHOBOS_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_HTML)
 
-phobos-release : ${PHOBOS_LATEST_FILES_GENERATED} $(DMD_LATEST) $(STD_DDOC) \
+phobos-release : ${PHOBOS_FILES_GENERATED} $(DMD) $(STD_DDOC) \
 		druntime-release dmd-release
-	$(MAKE) --directory=$(PHOBOS_LATEST_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_LATEST_HTML)
+	$(MAKE) --directory=$(PHOBOS_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_RELEASE_HTML)
 
 phobos-prerelease-verbatim : ${PHOBOS_FILES_GENERATED} ${DOC_OUTPUT_DIR}/phobos-prerelease/index.verbatim
 ${DOC_OUTPUT_DIR}/phobos-prerelease/index.verbatim : verbatim.ddoc \
