diff --git a/dlang.org.ddoc b/dlang.org.ddoc
index 5cef982..5dfcb7d 100644
--- a/dlang.org.ddoc
+++ b/dlang.org.ddoc
@@ -214,7 +214,6 @@ $(DIVID cssmenu, $(UL
     $(MENU https://tour.dlang.org, Learn)
     $(MENU_W_SUBMENU_LINK $(ROOT_DIR)documentation.html, Documentation)
       $(NAVIGATION_DOCUMENTATION)
-    $(MENU $(ROOT_DIR)download.html, Downloads)
     $(MENU https://code.dlang.org, Packages)
     $(MENU_W_SUBMENU_LINK $(ROOT_DIR)community.html, Community)
       $(NAVIGATION_COMMUNITY)
diff --git a/posix.mak b/posix.mak
index 2c380a8..56fbd5a 100644
--- a/posix.mak
+++ b/posix.mak
@@ -232,7 +232,7 @@ $(DOC_OUTPUT_DIR)/dmd-%.verbatim : %.ddoc dcompiler.dd verbatim.ddoc $(DMD)
 # Rulez
 ################################################################################
 
-all : docs html
+all : html druntime-release phobos-release d.tag
 
 docs : dmd-prerelease phobos-prerelease druntime-prerelease druntime-release \
   phobos-release apidocs-release apidocs-prerelease
@@ -258,9 +258,9 @@ ${GENERATED}/${LATEST}.ddoc :
 	mkdir -p $(dir $@)
 	echo "LATEST=${LATEST}" >$@
 
-${GENERATED}/modlist-${LATEST}.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR)-$(LATEST) $(PHOBOS_DIR)-$(LATEST)
+${GENERATED}/modlist-${LATEST}.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR) $(PHOBOS_DIR)
 	mkdir -p $(dir $@)
-	$(STABLE_RDMD) modlist.d $(DRUNTIME_DIR)-$(LATEST) $(PHOBOS_DIR)-$(LATEST) $(MOD_EXCLUDES_RELEASE) >$@
+	$(STABLE_RDMD) modlist.d $(DRUNTIME_DIR) $(PHOBOS_DIR) $(MOD_EXCLUDES_RELEASE) >$@
 
 ${GENERATED}/modlist-prerelease.ddoc : modlist.d ${STABLE_DMD} $(DRUNTIME_DIR) $(PHOBOS_DIR)
 	mkdir -p $(dir $@)
@@ -353,8 +353,8 @@ ${DMD_DIR} ${DRUNTIME_DIR} ${PHOBOS_DIR} :
 # dmd compiler, latest released build and current build
 ################################################################################
 
-$(DMD) : ${DMD_DIR}
-	${MAKE} --directory=${DMD_DIR}/src -f posix.mak -j 4
+#$(DMD) : ${DMD_DIR}
+#	${MAKE} --directory=${DMD_DIR}/src -f posix.mak -j 4
 
 $(DMD_REL) : ${DMD_DIR}-${LATEST}
 	${MAKE} --directory=${DMD_DIR}-${LATEST}/src -f posix.mak -j 4
@@ -373,9 +373,9 @@ druntime-prerelease : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC_PRE)
 		DOCDIR=${DOC_OUTPUT_DIR}/phobos-prerelease \
 		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC_PRE))"
 
-druntime-release : ${DRUNTIME_DIR}-${LATEST} $(DMD_REL) $(STD_DDOC)
-	${MAKE} --directory=${DRUNTIME_DIR}-${LATEST} -f posix.mak target doc \
-	  DMD=$(DMD_REL) \
+druntime-release : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC)
+	${MAKE} --directory=${DRUNTIME_DIR} -f posix.mak target doc \
+	  DMD=$(DMD) \
 	  DOCDIR=${DOC_OUTPUT_DIR}/phobos \
 		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC))"
 
@@ -400,12 +400,12 @@ phobos-prerelease : ${PHOBOS_DIR} $(STD_DDOC_PRE) druntime-prerelease
 	  STDDOC="$(addprefix `pwd`/, $(STD_DDOC_PRE))" \
 	  DOC_OUTPUT_DIR=${DOC_OUTPUT_DIR}/phobos-prerelease html -j 4
 
-phobos-release : ${PHOBOS_DIR}-${LATEST} $(DMD_REL) $(STD_DDOC) \
+phobos-release : ${PHOBOS_DIR} $(DMD) $(STD_DDOC) \
 		druntime-release
-	${MAKE} --directory=${PHOBOS_DIR}-${LATEST} -f posix.mak -j 4 \
+	${MAKE} --directory=${PHOBOS_DIR} -f posix.mak -j 4 \
 	  html \
-	  DMD=$(DMD_REL) \
-	  DRUNTIME_PATH=${DRUNTIME_DIR}-${LATEST} \
+	  DMD=$(DMD) \
+	  DRUNTIME_PATH=${DRUNTIME_DIR} \
 	  DOC_OUTPUT_DIR=${DOC_OUTPUT_DIR}/phobos \
 	  STDDOC="$(addprefix `pwd`/, $(STD_DDOC))"
 
diff --git a/std_navbar-release.ddoc b/std_navbar-release.ddoc
index 7899a22..42af129 100644
--- a/std_navbar-release.ddoc
+++ b/std_navbar-release.ddoc
@@ -4,7 +4,6 @@ $(SUBNAV_TEMPLATE
         $(H2 Library Reference)
         $(P $(SPANC smallprint, version $(LATEST) $(SPANC separator, $(BR))
             switch to
-            $(LINK2 ../phobos-prerelease/index.html, pre-release). $(BR)
             $(LINK2 ../library/index.html, preview) beta library reference.
             )
         )
