--- D-Programming-Language-phobos-9869e60/posix.mak	2011-04-12 16:15:54.698006265 -0600
+++ D-Programming-Language-phobos-9869e60/posix.mak	2011-04-12 16:20:05.317658490 -0600
@@ -45,7 +45,7 @@
 endif
 
 # Configurable stuff that's rarely edited
-DRUNTIME_PATH = ../druntime
+DRUNTIME_PATH = ../D-Programming-Language-druntime-715b0be/
 ZIPFILE = phobos.zip
 ROOT_OF_THEM_ALL = generated
 ROOT = $(ROOT_OF_THEM_ALL)/$(OS)/$(BUILD)/$(MODEL)
@@ -88,7 +88,7 @@
 		DMD = ssh 206.125.170.138 "cd code/dmd/phobos && dmd"
 		CC = ssh 206.125.170.138 "cd code/dmd/phobos && dmc"
 	else
-		DMD = dmd
+		DMD = ../D-Programming-Language-dmd-df42e28/src/dmd
 		ifeq ($(OS),win32)
 			CC = dmc
 		else
